compiler+runtime/src/cpp/main.cpp

```diff
@@ -124,27 +121,40 @@ namespace jank
       dynamic_call(__rt_ctx->in_ns_var->deref(), make_box<obj::symbol>(opts.target_module));
     }
 
-    /* By default, RL will do tab completion for files. We disable that here. */
-    rl_bind_key('\t', rl_insert);
+    auto const get_prompt([&](native_persistent_string const &suffix) {
```

Comment:
No need for the `[&]` capture here, I think. `__rt_ctx` is global, `suffix` is a param.